### ISSUE TITLE
Add config options for image and video target resolutions

### ISSUE THREAD

**Issue by Neriderc** [zusam/zusam#81]
Currently the resizing of videos and photos is to a maximum of 2048x2048 for images and 720p for videos. We should make these config options, including the ability to disable the conversion.

For example, we could have:
IMAGE_MAX_WIDTH="2048"
IMAGE_MAX_HEIGHT="2048"
VIDEO_RESOLUTION="720p"

Setting these to "-1" (or ""?) could disable conversion.

**Comment by nrobinaubertin** [comment]
Good idea. I think that values should be empty to avoid scaling on that axis (conversion could still happen in my opinion, depending on the file format).  
I think that video resolution should be a number without the `p` suffix. That way we can parse it into an integer if need be.
